build/typescript-model/generate.sh

```diff
@@ -53,12 +53,49 @@ EOF
     export OPENAPI_GENERATOR_COMMIT="v6.3.0"
     bash $WORK_DIR/gen/openapi/typescript.sh $WORK_DIR/typescript-models $WORK_DIR/config.sh
 
-    apply_sed 's/\"name\": \".*\"/"name": "@devfile\/api"/g' $WORK_DIR/typescript-models/package.json
-    apply_sed 's/\"description\": \".*\"/"description": "Typescript types for devfile api"/g' $WORK_DIR/typescript-models/package.json
-    apply_sed 's/\"repository\": \".*\"/"repository": "devfile\/api"/g' $WORK_DIR/typescript-models/package.json
-    apply_sed 's/\"license\": \".*\"/"license": "Apache-2.0"/g' $WORK_DIR/typescript-models/package.json
-    apply_sed 's/\"@types\/bluebird\": \".*\"/"@types\/bluebird": "3.5.21"/g' $WORK_DIR/typescript-models/package.json
```

Comment:
Do we know if it's okay to remove
```
    apply_sed 's/\"@types\/bluebird\": \".*\"/"@types\/bluebird": "3.5.21"/g' $WORK_DIR/typescript-models/package.json
```
?

Reply:
I could have deleted it by accident.
I will double check it.

Reply:
I recalled why did I remove it at all, there was no dependency on `bluebird` in `package.json`.

To check, go to https://www.npmjs.com/package/@devfile/api?activeTab=code and open existing `package.json`